tests/edu/oregonstate/cs361/battleship/BattleshipModelTest.java

```diff
@@ -91,17 +91,18 @@ void shootAtComputer() {
         assertEquals(6, model.computerHits.get(1).getAcross());
         assertEquals(8, model.computerHits.get(1).getDown());
 
-        model.shootAtComputer(4,4) ;
-        assertEquals(4, model.computerHits.get(2).getAcross());
-        assertEquals(4, model.computerHits.get(2).getDown());
-
         model.shootAtComputer(7,3) ;
```

Comment:
in the future, we might want the change the name of this since it seems like we are shooting at a computer rather than the ship that belongs to the computer